packages/modular-scripts/src/program.ts

```diff
@@ -124,11 +124,20 @@ interface CLITestOptions extends TestOptions {
 
 program
   .command('test [regexes...]')
+  .option(
+    '--ancestors',
+    'Additionally run tests for workspaces that depend on workspaces that have changed',
+    false,
+  )
   .option(
     '--debug',
     'Setup node.js debugger on the test process - equivalent of setting --inspect-brk on a node.js process',
     false,
   )
+  .option(
+    '--changed <branch>',
```

Comment:
To allow for the future state of having `--changed` on its own to use the default / PR head, should this be a `boolean` with another option of `--compare-branch`

Reply:
Agree, it'd be clearer.

Reply:
Done